feature-libs/commerce-quotes/components/commerce-quotes-list/commerce-quotes-list.component.html

```diff
@@ -36,10 +36,10 @@ <h3 id="header">{{ 'commerceQuotesList.header' | cxTranslate }}</h3>
               {{ 'commerceQuotesList.quoteName' | cxTranslate }}
             </th>
             <th scope="col">
-              {{ 'commerceQuotesList.quoteId' | cxTranslate }}
+              {{ 'commerceQuotesCommons.quoteId' | cxTranslate }}
             </th>
             <th scope="col">
-              {{ 'commerceQuotesList.quoteStatus' | cxTranslate }}
+              {{ 'commerceQuotesCommons.quoteStatus' | cxTranslate }}
```

Comment:
Related to the above comments - let's go with simple `commons` which will result in a more readable result `commerceQuotes.commons.xyz`